### ISSUE TITLE
Accessibility: Redundant WAI-ARIA attribute

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#443]
**Description**
This issue appears when an HTML5 element that inherently gives a role to a section also has a "role" attribute added to it. This makes the "role" attribute redundant and it should be removed.

This particular issue is showing up on the Form elements for exhibit contact forms and the feedback form.

**Solution**
Remove `role="form"` from all `<form>` tags
